File: 1.py
f = open("input/1.txt", "r")
lines = []
for line in f.readlines():
    lines.append(line)

sum = 0
for line in lines:
    integer = ""
    for c in line:
        if (c.isdigit()):
            integer+= c
            break
    for c in line[::-1]:
        if (c.isdigit()):
            integer += c
            break
    sum += int(integer)
print(sum)
# print(lines)

File: 1.cpp
// reading a text file
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main () {
  string line;
  ifstream myfile ("input/1.txt");
  string lines[1000];
  if (myfile.is_open())
  {
    int i = 0;
    while ( getline (myfile,line) )
    {
        // lines[i] = strcpy(line);
        i++;
    }
    myfile.close();
  }
  else cout << "Unable to open file"; 

  return 0;
}

